ceph/rbd/workflows/rbd.py

```diff
@@ -139,7 +145,8 @@ def create_single_pool_and_images(
     ceph_version,
     is_ec_pool,
     is_secondary,
-    create_image_parallely,
+    create_image_parallely=False,
+    do_not_create_image=False,
```

Comment:
what better name do you suggest? This name felt as relevant as it can get for me.
That's how it works. As you can see its default value is false. If its specified as true while calling then it won't create images.